packages/wrangler/src/__tests__/docs.test.ts

```diff
@@ -0,0 +1,122 @@
+import { http, HttpResponse } from "msw";
+import { afterEach, beforeEach, describe, test } from "vitest";
+import openInBrowser from "../open-in-browser";
+import { mockConsoleMethods } from "./helpers/mock-console";
+import { msw } from "./helpers/msw";
+import { runWrangler } from "./helpers/run-wrangler";
+
+// NOTE: in production builds we "esbuild define" Algolia constants as globals
+// but in tests we have to attach mocks values to the globalThis object.
+
+// eslint-disable-next-line @typescript-eslint/no-namespace
+declare module globalThis {
+	let ALGOLIA_APP_ID: string | undefined;
+	let ALGOLIA_PUBLIC_KEY: string | undefined;
+}
+
+describe("wrangler docs", () => {
+	const std = mockConsoleMethods();
+
+	beforeEach(() => {
+		globalThis.ALGOLIA_APP_ID = "FAKE-ID";
+		globalThis.ALGOLIA_PUBLIC_KEY = "FAKE-KEY";
+
+		msw.use(
+			http.post<Record<string, never>, { params: string | undefined }>(
+				`*/1/indexes/developers-cloudflare2/query`,
+				async ({ request }) => {
+					const json = await request.json();
+					const params = new URLSearchParams(json.params);
+					return HttpResponse.json({
+						hits: [
+							{
+								url: "FAKE_DOCS_URL?" + params.get("query"),
+							},
+						],
+					});
+				},
+				{ once: true }
+			)
+		);
+	});
+
+	afterEach(() => {
+		delete globalThis.ALGOLIA_APP_ID;
+		delete globalThis.ALGOLIA_PUBLIC_KEY;
+	});
+
+	test("--help", async ({ expect }) => {
+		const result = runWrangler("docs --help");
+
+		await expect(result).resolves.toBeUndefined();
+		expect(std.out).toMatchInlineSnapshot(`
+			"wrangler docs [command..]
+
+			📚 Open Wrangler's command documentation in your browser
+
+
+			POSITIONALS
+			  command  Enter the wrangler command you want to know more about  [array] [default: []]
```

Comment:
Does it makes sense to pass an array?

Reply:
The change that was made last year to convert this to a "I'm feeling lucky" search means that you can pass in multiple items. The fact it says array here, is really just an artifact of how yargs trees variadic positional args.

What we could consider doing is changing the docs to reflect the fact that this is no longer just looking for a `command` but search terms.

Reply:
> What we could consider doing is changing the docs to reflect the fact that this is no longer just looking for a command but search terms.

What about doing that in this PR while we are fixing the command?